cfme/tests/integration/test_ipa_external.py

```diff
@@ -6,6 +6,8 @@
 from utils.conf import cfme_data
 from utils.providers import setup_a_provider
 
+pytestmark = [pytest.mark.uncollect]
```

Comment:
It would have to be guaranteed that the virtual machine has correct fqdn, which we cannot if the appliance is running on openstacks, which will give it a private address.